proxy-libs/helioslambda/lambda.go

```diff
@@ -2,21 +2,44 @@ package helioslambda
 
 import (
 	"context"
+	"encoding/json"
 
 	"github.com/aws/aws-lambda-go/lambda"
 	"go.opentelemetry.io/contrib/instrumentation/github.com/aws/aws-lambda-go/otellambda"
 	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/propagation"
 	"go.opentelemetry.io/otel/sdk/trace"
 )
 
 var InstrumentedSymbols = [...]string{"Start", "StartWithContext", "StartWithOptions"}
 
+type HttpHeaders struct {
+	Headers map[string]string `json:"headers"`
+}
+func heliosEventToCarrier (eventJSON []byte) propagation.TextMapCarrier {
+	var headers HttpHeaders
+	err := json.Unmarshal(eventJSON, &headers)
+	if err != nil {
+		return propagation.HeaderCarrier{"": []string{""}}
```

Comment:
you can remove this err != nil, the return in the last statement takes care of that

Reply:
but I'm not sure it will not throw me if I'll try to acsess headers

Reply:
What I'm saying is you can just check if err == nil, else return propagation.HeaderCarrier{"": []string{""}}